### DISCUSSION TITLE
Downloader not working for me on Edge and Chrom

### DISCUSSION THREAD

**Discussion by autonoes** [elementdavv/internet_archive_downloader#61] [General]
Hello! Yesterday the downloader was working fine for me (on Edge), but now it doesn't seem to be. The usual download and quality buttons pop up under my borrowed books, but clicking them isn't doing anything at all. I tested it on Chrome aswell, didn't work there either. Is anyone else experiencing this?

**Comment by elementdavv** [comment]
upgrade to v1.0.2

**Comment by AnxiousPleaseHelp** [comment]
> Hello! Yesterday the downloader was working fine for me (on Edge), but now it doesn't seem to be. The usual download and quality buttons pop up under my borrowed books, but clicking them isn't doing anything at all. I tested it on Chrome aswell, didn't work there either. Is anyone else experiencing this?

Have you tried Firefox? That works pretty damn well!

**Comment by norm09** [comment]
I was having the same problem but try this: open the book in Internet Archive, click on the Internet Archive Downloader extension in your extensions, the control panel should appear, at the very top Archive is next to HathiTrust, click "Archive". Your browser should navigate to the Internet Archive homepage. You can then search for your book and the button should work!

**Comment by AnxiousPleaseHelp** [comment]
> Hello! Yesterday the downloader was working fine for me (on Edge), but now it doesn't seem to be. The usual download and quality buttons pop up under my borrowed books, but clicking them isn't doing anything at all. I tested it on Chrome aswell, didn't work there either. Is anyone else experiencing this?

Did you ever get this working?

Try Orion Browser for iOS. It supports Firefox extensions!
